.azure/gpu-tests.yml

```diff
@@ -111,14 +117,7 @@ jobs:
         PL_RUN_CUDA_TESTS: "1"
       displayName: 'Testing: PyTorch standalone tests'
       timeoutInMinutes: "35"
-
-    - bash: bash run_standalone_tasks.sh
-      workingDirectory: tests/tests_pytorch
-      env:
-        PL_USE_MOCKED_MNIST: "1"
-        PL_RUN_CUDA_TESTS: "1"
-      displayName: 'Testing: PyTorch standalone tasks'
-      timeoutInMinutes: "10"
```

Comment:
This was an accident, no? Opened https://github.com/Lightning-AI/lightning/pull/14366

Reply:
All I did is I clicked on the commit and clicked "revert". So I don't know how this happened.